contracts/Vault.vy

```diff
@@ -190,6 +195,16 @@ def __init__(
     self.depositLimit = MAX_UINT256  # Start unlimited
     self.lastReport = block.timestamp
     self.activation = block.timestamp
+    # EIP-712
+    self.DOMAIN_SEPARATOR = keccak256(
+        concat(
+            DOMAIN_TYPE_HASH,
+            keccak256(convert("Yearn Vault", Bytes[64])),
```

Comment:
Not sure this even changes anything lol, since encoding is via length not size
```suggestion
            keccak256(convert("Yearn Vault", Bytes[11])),
```